scss/civicrm/contact/pages/_new-individual.scss

```diff
@@ -248,4 +248,30 @@
       }
     }
   }
+
+  #contactDetails {
+    tr:not([id]) {
+      // Makes labels bold. We cannot use a specific selector because
+      // labels are just text inside <td>s. We also ignore <tr>s with "id"
+      // attributes because those definitely do not contain labels
+      font-weight: $crm-font-weight-bold;
+
+      // Discarding bold font for links
+      a {
+        font-weight: $crm-font-weight-normal;
+      }
+    }
+  }
+
+  .email-signature > .collapsible-title {
+    font-size: $font-size-base;
+    font-weight: $crm-font-weight-bold;
+  }
+
+  // Styling Postcode Lookup extension input
+  #addressLookup input {
+    @include input-text-bootstrap();
+    // Makes the placeholder fully visible
+    min-width: 220px;
```

Comment:
Do you think we can document this in a variable?

Reply:
That would be a bit confusing, because this number just matches the placeholder text length inside the input, so it won't be reused.